### DISCUSSION TITLE
Can be set as subfolder in Nginx Proxy Manager?

### DISCUSSION THREAD

**Discussion by PrzemekSkw** [ckulka/baikal-docker#127] [General]
Hello, I use NPM for expose my services. I install Baikal now for the first time and wonder if it's possible to use it in subfolder? Or do I have to get new domain?
Regards.

**Comment by ckulka** [comment]
I'm not familiar with NPM, but it should be possible if you do two things:

1. Specify the subfolder as `base_uri` in the [`baikal.yaml` file](https://github.com/sabre-io/Baikal/blob/31e7d54f5ad4e85db764970c8e3bf83c2a79dbd3/config/baikal.yaml.dist#L11)
2. Forward all HTTP requests for that subpath to the Baikal container

Especially for 2. you'd have to see how that works in your setup, e.g. if you have Traefik, Nginx, or something else where you need to configure the routing.

Hope this helps!
